src/studio-header/Header.jsx

```diff
@@ -156,7 +166,7 @@ const Header = ({
     logo: config.LOGO_URL,
     logoAltText: 'Studio edX',
     siteName: 'edX',
-    logoDestination: config.STUDIO_BASE_URL,
+    logoDestination: process.env.ENABLE_NEW_HOME_PAGE === 'true' ? '/home' : config.STUDIO_BASE_URL,
```

Comment:
Have we decided that studio home is going to live in the "course authoring MFE"? if so, that furthers my case that we should have one mfe.

Reply:
I don't believe that we have come to a formal decision on this. It is just easiest at the moment to set it in course-authoring.